### ISSUE TITLE
Kotlin know supports CBOR 

### ISSUE THREAD

**Issue by StokeMasterJack** [cbor/cbor.github.io#77]
Kotlin know supports CBOR as part of kotlinx-serialization:

[https://kotlin.github.io/kotlinx.serialization/kotlinx-serialization-cbor/kotlinx-serialization-cbor/kotlinx.serialization.cbor/index.html](https://kotlin.github.io/kotlinx.serialization/kotlinx-serialization-cbor/kotlinx-serialization-cbor/kotlinx.serialization.cbor/index.html)

**Comment by Whathecode** [comment]
Indeed: [`kotlinx.serialization`](https://github.com/Kotlin/kotlinx.serialization/tree/master?tab=readme-ov-file) supports CBOR in Kotlin

And, there is also [a third-party CBOR integration](https://github.com/Kotlin/kotlinx.serialization/blob/master/formats/README.md#other-community-supported-formats) with `kotlinx.serialization`: https://github.com/L-Briand/obor
